### ISSUE TITLE
Hey,

### ISSUE THREAD

**Issue by Rizwan2613** [QwenLM/Qwen2-VL#511]

**Comment by data-ant** [comment]
You mean, when you use qwen2-vl for detection, the output bounding boxes is inaccurate, right?

**Comment by Rizwan2613** [comment]
yes. I'm working with documents (DocVQA) but whenever I want grounding, it gives wrong bounding box.

**Comment by data-ant** [comment]
我已收到，谢谢。
                      祝您诸事顺利!

**Comment by Cece1031** [comment]
Did you find how to get the attention map out of qwen2vl? Thank you very much!

**Comment by data-ant** [comment]
我已收到，谢谢。
                      祝您诸事顺利!

**Comment by data-ant** [comment]
我已收到，谢谢。
                      祝您诸事顺利!
